README.md

```diff
@@ -1,7 +1,9 @@
 
 # National and subnational estimates of the time-varying reproduction number for Covid-19
 
-This repository contains estimates of the time-varying reproduction number for every country in the world listed in the ECDC Covid-19 data source and subnational estimates for 9 countries. Summarised estimates can be found in `national/cases/summary` and `national/deaths/summary` (based on cases and deaths respectively). Estimates for each country can be found in `national/cases/national` and `national/deaths/national`. Subnational estimates can be found in the relevant country folder (`subnational/country`) with the same folder structure as for the national estimates. Estimates are generated using [`{EpiNow2}`](https://epiforecasts.io/EpiNow2/) and presented on [epiforecasts.io/covid](https://epiforecasts.io/covid) (which also outlines the method used). An [interactive visualisation](https://hamishgibbs.github.io/rt_vis/) is currently under development.
+This repository contains estimates of the time-varying reproduction number for every country in the world listed in the ECDC Covid-19 data source and subnational estimates for 9 countries. Summarised estimates can be found in `data/results/national/cases/summary` and `data/results/national/deaths/summary` (based on cases and deaths respectively). Estimates for each country can be found in `data/results/national/cases/national` and `data/results/national/deaths/national`. Subnational estimates can be found in the relevant country folder (`data/results/subnational/country`) with the same folder structure as for the national estimates. Estimates are generated using [`{EpiNow2}`](https://epiforecasts.io/EpiNow2/) and presented on [epiforecasts.io/covid](https://epiforecasts.io/covid) (which also outlines the method used). An [interactive visualisation](https://hamishgibbs.github.io/rt_vis/) is currently under development.
+
+WARNING: We are in a transition phase - the results can be generated in two locations. Only results in /data/results will be committed and production will shift to this location. Data will be output to this location by using the `-d` flag. For those who require data in the old location you can (for the time being) run without the -d flag. This will be deprecated!
```

Comment:
Suggest drop notes about estimate generation as the intro paragraph should strictly target data users (as per #107) with developer info elsewhere + linked.

Reply:
Also this may now have a merge issue due to #104 and #90?

Reply:
I'll do a rebase of the latest master and see what happens